### ISSUE TITLE
Issue in AWS specific AWS config since Jan 6th, 2021

### ISSUE THREAD

**Issue by JeffreyAtQ** [SeleniumHQ/docker-selenium#1175]

**Comment by diemol** [comment]
Does this only happen when you use AWS? 

What happens when you use the docker images in a different environment?

What test script can be used to reproduce this?

Can you please provide the container logs as well?

**Comment by JeffreyAtQ** [comment]
So, when we run the selenium tests locally and without any nodes on our windows 10 VM's, everything works as expected, but that's not using the Grid at all. We haven't had time set up a separate environment outside of AWS, we've been performing some archeology on work done by a previous colleague as a result of this issue coming up and only just today figured out the environment variable that could be used to roll back the docker image in our project execution.

In short, working on getting answers (and file access) to those questions ASAP, will update when I do.
